### ISSUE TITLE
Crash on 1.8

### ISSUE THREAD

**Issue by hangingmoon** [Pathos0925/JWEM#16]
The game crashes at startup during the copyright screen. Was working on previous versions of the game.

**Comment by LPGozzzi** [comment]
Yes, as said above, it doesn't let you go past the first (copyright) screen about Universal etc and in conclusion the game crashes.

**Comment by Pathos0925** [comment]
The modkit does not work with this latest version of Jurassic World Evolution. You'll need to either remove the modkit (by deleting the files that you copied into the game directory) or use a previous version of the game. Deleting the dxgi.dll / d3d11.dll should be enough, but you can also delete the other files associated with the devkit as well. If you happened to delete the wrong files, you can reinstall them with the "Verify Integrity" option: https://support.steampowered.com/kb_article.php?ref=2037-QEUH-3335

**Comment by james-creator** [comment]
Is this something that can be fixed or is the concept of this mod no longer viable?

**Comment by Nabukiller** [comment]
Hey "Pathos0925" Can i pay you to get this updated to work again for this update?

**Comment by Nabukiller** [comment]
Will be an update for us? Please :D
